### ISSUE TITLE
TypeScript error when using props mounting option in `@cypress/vue`

### ISSUE THREAD

**Issue by sodatea** [cypress-io/cypress#17480]
### Current behavior

When type checking the following code:
```js
import { mount } from '@cypress/vue'
import HelloWorld from '../HelloWorld.vue'

mount(HelloWorld, { props: { msg: 'Hello Cypress' } })
```

TypeScript throws the following error:

```
src/components/__tests__/HelloWorld.spec.ts:6:25 - error TS2322: Type '{ msg: string; }' is not assignable to type 'VNodeProps & { __v_isVNode?: undefined; [Symbol.iterator]?: undefined; } & Record<string, any> & { $: ComponentInternalInstance; $data: {}; ... 11 more ...; $watch(source: string | Function, cb: Function, options?: WatchOptions<...> | undefined): WatchStopHandle; } & ... 4 more ... & ComponentCustomProperties'.
  Type '{ msg: string; }' is missing the following properties from type '{ $: ComponentInternalInstance; $data: {}; $props: Partial<{}> & Omit<Readonly<{ msg?: unknown; } & { msg: String; } & {}> & VNodeProps & AllowedComponentProps & ComponentCustomProps, never>; ... 10 more ...; $watch(source: string | Function, cb: Function, options?: WatchOptions<...> | undefined): WatchStopHandle; }': $, $data, $props, $attrs, and 10 more.

6     mount(HelloWorld, { props: { msg: 'Hello Cypress' } })
                          ~~~~~
```

### Desired behavior

No error is thrown, works as the `mount` method in `@vue/test-utils`.

### Test code to reproduce

https://github.com/sodatea/cypress-type-error

```sh
git clone https://github.com/sodatea/cypress-type-error.git
cd cypress-type-error
pnpm i
pnpm typecheck
```

### Cypress Version

8.0.0

### Other

Using `@cypress/vue` 3.0.0-beta.3 as it's the latest release.

Though, there's a `@cypress/vue` 3.0.1 released previously on npm. I have to pin the version to use beta.3. This should be fixed too.

**Comment by elevatebart** [comment]
Both have been released in `@cypress/vue@3.0.3`
Please close once you confirm.

Thank you.

**Comment by sodatea** [comment]
I can confirm it's fixed now. Thanks!
